packages/web-api/src/types/response/ChatStopStreamResponse.ts

```diff
@@ -0,0 +1,9 @@
+import type { WebAPICallResult } from '../../WebClient';
+export type ChatStopStreamResponse = WebAPICallResult & {
```

Comment:
suggestion: Looking at the reference, we'll also want a `message` object type that includes a `streaming_state` string.